### ISSUE TITLE
Suspend only cronjobs?

### ISSUE THREAD

**Issue by AmitBaranes** [kube-green/kube-green#165]
is it possible to suspend only cronjobs? I've tested the following template but no luck -

```yaml
apiVersion: kube-green.com/v1alpha1
kind: SleepInfo
metadata:
  name: working-hours
spec:
  weekdays: "*"
  sleepAt: "16:30"
  wakeUpAt: "10:00"
  timeZone: "Asia/Jerusalem"
  suspendCronJobs: true
  excludeRef:
    - apiVersion: "apps/v1"
      kind: Deployment
      name: "*"
```

**Comment by davidebianchi** [comment]
No, it is not possible at the moment. It could be possible to add a boolean to enable or disable also the Deployment (as for the CronJobs).
What do you think about it?
What is the use case you need it for?

**Comment by AmitBaranes** [comment]
sounds good to me.
We have multiple environments under our cluster, each environment set up with the relevant microservices project ( backend - API  - etc.. ) most of them have cronjobs that do not need to run on every environment. 
my goal is the suspend them using Kube-green and if a developer needs to enable a specific cronjob this can be done manually.

**Comment by AmitBaranes** [comment]
Any ETL?

**Comment by davidebianchi** [comment]
Ok, clear! Thanks for the use case. It should be quite easy to add, I try to add it in the next few days

**Comment by AmitBaranes** [comment]
Thanks for the quick response, I appreciate it!

**Comment by AmitBaranes** [comment]
any update? :)

**Comment by davidebianchi** [comment]
I already started the implementation, but it isn't finished yet.

**Comment by AmitBaranes** [comment]
Thank you, an update here will help us a lot !

**Comment by davidebianchi** [comment]
Hi @AmitBaranes, the feature is released in 0.4.0
